haskell/test-common/Lorentz/Contracts/Test/Common.hs

```diff
@@ -0,0 +1,67 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+module Lorentz.Contracts.Test.Common
```

Comment:
If we run nettest test-suite on CI, we don't need to include this scenario into two test-suites, it's sufficient to put it into only one (like we did in `morley-ledgers-test`).

Reply:
Well, maybe something should still be in test-common, but you definitely put much more than necessary. And `scTransferScenario` shouldn't be there.